### ISSUE TITLE
Resolution REGION is not supported?

### ISSUE THREAD

**Issue by emako** [haplokuon/netDxf#123]
I need some help about DXF code of REGION entity.
But REGION is not supported?

like it :
REGION
  5
AAD8
330
1F
100
AcDbEntity
  8
47474747
 62
     7
420
 16777215
100
AcDbModelerGeometry
 70
     1
  1
mnmoo lg m mi          
  1
ni ^ *+0;:,4 ^ *+0\^ [ ng ^ LR nmqoqoqjgmm QK o  
  1
n fqfffffffffffffffj:rooh n:rono 
  1
>,27:>;:- {rn rn _nm mnmqoqoqjgmm |
  1
=0;& {m rn {rn {l {rn {rn |
  1
-:9@)+r:&:r>++-6= {rn rn {rn {rn {n {k {j |
  1
3*2/ {i rn {rn {rn {h {n |
  1
:&:@-:961:2:1+ {rn rn _j 8-6;  n _l +-6 n _k ,*-9 o _l >;5 o _k 8->; o _f /0,+<7:<4 o _k ,+03 oqfjohhlnhfjlnofhkn _k 1+03 lo _k ;,63 o _g 93>+1:,, o _h /6'>-:> o _k 72>' o _i 8-6;>- o _j 28-6; looo _j *8-6; o _j )8-6; o _no :1;@96:3;, |
  1
):-+:'@+:2/3>+: {rn rn l o n g |
  1
-:9@)+r:&:r>++-6= {rn rn {rn {rn {l {k {j |
  1
,7:33 {g rn {rn {rn {rn {f {rn {l |
  1
-:9@)+r:&:r>++-6= {rn rn {rn {rn {h {k {j |
  1
9><: {no rn {rn {rn {nn {h {rn {nm 90-(>-; ;0*=3: 0*+ |
  1
92:,7r:&:r>++-6= {rn rn {nl {rn {f |
  1
300/ {rn rn {rn {rn {nk {f |
  1
/3>1:r,*-9><: {rn rn {rn gjlqnjhloklknoiljk mokqgjioioijomgnjn o o o n n o o 90-(>-;@) V V V V |
  1
-:9@)+r:&:r>++-6= {rn rn {rn {no {f {k {j |
  1
<0:;8: {rn rn {rn {nj {ni {rn {nh -:):-,:; {nn {rn |
  1
<0:;8: {rn rn {rn {ng {nk {rn {nf -:):-,:; {nn {rn |
  1
<0:;8: {rn rn {rn {nk {ng {rn {mo -:):-,:; {nn {rn |
  1
:;8: {mn rn {rn {mm o {ml no {nk {mk 90-(>-; _h *1410(1 |
  1
<0:;8: {rn rn {rn {ni {nj {rn {mj -:):-,:; {nn {rn |
  1
:;8: {mi rn {rn {mh o {mm j {nj {mg 90-(>-; _h *1410(1 |
  1
:;8: {mf rn {rn {ml o {lo j {ni {ln 90-(>-; _h *1410(1 |
  1
/+36,+r:&:r>++-6= {rn rn {rn {rn {nh |
  1
):-+:' {rn rn {rn {nh o {lm |
  1
):-+:' {rn rn {rn {nh n {ll |
  1
,+->687+r<*-): {rn rn {rn gjoqijhloklknoiljk nffqgjioioijomgnim o o n o V V |
  1
:;8: {lk rn {rn {lo o {mh no {ng {lj 90-(>-; _h *1410(1 |
  1
/+36,+r:&:r>++-6= {rn rn {rn {rn {nf |
  1
):-+:' {rn rn {rn {mj n {li |
  1
,+->687+r<*-): {rn rn {rn gjjqijhloklknoiljk nffqgjioioijomgnim o rn o o V V |
  1
/+36,+r:&:r>++-6= {rn rn {rn {rn {mo |
  1
):-+:' {rn rn {rn {mo n {lh |
  1
,+->687+r<*-): {rn rn {rn gjoqijhloklknoiljk mofqgjioioijomgnim o n o o V V |
  1
/061+ {rn rn {rn gjoqijhloklknoiljk nffqgjioioijomgnim o |
  1
/061+ {rn rn {rn gjoqijhloklknoiljk mofqgjioioijomgnim o |
  1
/+36,+r:&:r>++-6= {rn rn {rn {rn {mj |
  1
,+->687+r<*-): {rn rn {rn gjjqijhloklknoiljk mofqgjioioijomgnim o o rn o V V |
  1
/061+ {rn rn {rn gjjqijhloklknoiljk nffqgjioioijomgnim o |
  1
/061+ {rn rn {rn gjjqijhloklknoiljk mofqgjioioijomgnim o |
  0
LINE
  5
AAD9
330
1F
100
AcDbEntity
  8
47474747
 62
     7
420
 16777215
100
AcDbLine
 10
855.6573043410635
 20
204.8560606502816
 30
0.0
 11
855.6573043410635
 21
208.8560606502816
 31
0.0
  0
HATCH
  5
AADA
330
1F
100
AcDbEntity
  8
47474747
 62
     7
420
 16777215
100
AcDbHatch
 10
0.0
 20
0.0
 30
0.0
210
0.0
220
0.0
230
1.0
  2

**Comment by haplokuon** [comment]
No, REGION as SURFACE and 3DSOLID entities are not supported, they never will be. They depend on proprietary data  that Autodesk never made public. See all that gibberish under multiple code 1 entries, well, that is the actual description of the entity.
